Tiltfile

```diff
@@ -41,12 +61,18 @@ if os.getenv('KKA_DEPLOY_MINIMAL', 'false') == 'false':
         namespace='argocd',
     ))
 
+    valueFiles = []
+    values = ['revision=%s' % os.getenv('KKA_REPO_BRANCH')]
+    if os.path.exists('./manifests/bootstrap/overrides.yaml'):
```

Comment:
I think it's better to have something like `./bootstrap.local.yaml` on the root that is both easier to find and easier to delete in case. What do you think?

Reply:
Totally, this is something I wanted to discuss during today standup, we have to find a place and a name that works for everyone.

Reply:
but maybe it does make sense for development, let me think about it.